src/nitypes/waveform/__init__.py

```diff
@@ -80,10 +80,10 @@
 
 Timing objects are immutable, so you cannot directly set their properties:
 
->>> wfm.timing.sample_interval = dt.timedelta(seconds=10e-3)
+>>> wfm.timing.sample_interval = dt.timedelta(seconds=10e-3)  # doctest: +ELLIPSIS
 Traceback (most recent call last):
 ...
-AttributeError: property 'sample_interval' of 'Timing' object has no setter
+AttributeError: ...
```

Comment:
I originally used `# doctest: +SKIP` but that disables the test all of the time. Also, commenting the reason like `# doctest: +SKIP # pypy3.10` doesn't work because doctest errors about "invalid option #"